libs/dal/src/repositories/organization/organization.entity.ts

```diff
@@ -17,6 +17,8 @@ export class OrganizationEntity {
   partnerConfigurations?: IPartnerConfiguration[];
 }
 
+export type OrganizationDBModel = OrganizationEntity;
```

Comment:
I find weird that the DBModel and the Entity are the same in this case. Shouldn't the entity return with the `OrganizationId` for the `_id`?
Also `DbModel._id` should be of type `Types.ObjectId` for coherence with the rest of the changes done.

Reply:
The `DbModel._id` type is not required by the schema, it's created automatically, so the `OrganizationEntity` just has to have it defined as the `_id: string` as it does, and because of that the `OrganizationDBModel = OrganizationEntity` for this case.